core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Project.java

```diff
@@ -44,7 +44,7 @@
  * @author Rob Davey
  * @since 0.0.2
  */
-public interface Project extends Comparable<Project>, SecurableByProfile, Watchable, Nameable, Serializable {
+public interface Project extends Comparable<Project>, SecurableByProfile, Watchable, Nameable, Serializable, Attachable {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)